modules/FeaturedCategories/ui/CategoryImage.module.scss

```diff
@@ -0,0 +1,32 @@
+.imageContainer {
+    display: block;
+    height: 100%;
+}
+
+.placeholder {
+    background-color: var(--prezly-accent-color);
+}
+
+.image, .placeholder {
+    @include border-radius-s;
+
+    height: 151px;
```

Comment:
What an interesting number 🤔

Reply:
He took it from my Figma (which I've since rounded down) but he seems to be particularly attached to it 😅 - I say we change it to 150 as is making me twitch @kudlajz